packages/ui/brand/src/components/experimental/experimental.stories.tsx

```
@@ -288,19 +288,52 @@ export const BrandArc: Story = {
+        </div>
+        <style>{`
+          @keyframes brandArcFadeIn {
+            from { opacity: 0; }
+            to   { opacity: 1; }
+          }
+          @keyframes brandArcFadeOut {
+            from { opacity: 1; }
+            to   { opacity: 0; }
+          }
+        `}</style>
+        <svg key={cycle} width={size} height={size}>
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)